scripts/code-style.sh

```
@@ -22,14 +22,37 @@ FOLDERS_TO_FORMAT=("dags" "xlml")
+          fi
+        done
+      done \
+    | sort -u
+)"
+
+if [[ -z "${CHANGED_PY_FILES}" ]]; then
+  echo "[pre-push hook] no changed files detected between ${HEAD_SHA} and ${BASE_BRANCH}"
+  exit 1
+fi
+
+pyink ${CHANGED_PY_FILES} --pyink-indentation=2 --pyink-use-majority-quotes --line-length=80 --check --diff
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)